swagger.yaml

```diff
@@ -403,6 +452,13 @@ paths:
           description: Missing paramaters
 components:
   schemas:
+    InviteToken:
+      type: object
+      properties:
+        token:
+          type: integer
+          format: int64
+          example: 128372874
```

Comment:
Since is crucial data, I'd recommend to use random generated string (what we should have done everywhere to be honest)

Reply:
Okey